packages/theming/atlas/src/themesource/atlas_core/web/core/helpers/_navigation-bar.scss

```diff
@@ -16,12 +16,20 @@
     .region-topbar {
         .mx-navbar {
             background-color: $navtopbar-bg;
+            min-height: auto;
 
             ul.nav {
+                
+                > .mx-navbar-item{
+                    margin-left: $spacing-small;
+                }
                 /* Navigation item */
                 & > li.mx-navbar-item > a {
                     color: $navtopbar-color;
                     font-size: $navtopbar-font-size;
+                    line-height: 1.2;
```

Comment:
Don't you want to reuse the `$line-height-base` variable here?

Reply:
This $line-height-base has a value of 1.428571429 which doesnt comply with the new specifications from design. As this only affects the topbar, we could keep there